epregressions/runtests.py

```diff
@@ -446,6 +446,7 @@ def diff_text_files(file_a, file_b, diff_file):
         txt1_cleaned = []
         skip_strings = [
             "Program Version,EnergyPlus",
+            "Version,",
```

Comment:
Ignore these lines in text files, which affects the diff calculations for EIO and IDF files primarily.